lib_eio/path.mli

```diff
@@ -40,6 +40,9 @@ val ( / ) : 'a t -> string -> 'a t
 val pp : _ t Fmt.t
 (** [pp] formats a [_ t] as "<label:path>", suitable for logging. *)
 
+val size : _ t -> int
```

Comment:
This will be too small on 32-bit systems. We're using int63 for file offsets, so should probably use that here too.

Reply:
I was wondering, because we already take an `int` as `maxsize` in `Buf_read`, so are we actually supporting 32bit ?

Reply:
You can't load a 4GB file into memory in one go on a 32-bit system, so it's fine for the maximum buffer size to be an `int`. But you should still be able to find out how big it is and then load it in chunks.

(`size` returning an `int` would be OK for `Path.load`, but not necessarily for other uses)